src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss

```diff
@@ -1,7 +1,7 @@
 .wrapper {
   position: absolute;
   display: flex;
-  justify-content: flex-end;
```

Comment:
We style according to the desktop first rule. So move it to the `@include mobile` mixin. The `flex-end` value should stay for divs without `withBackButtonClass` for the desktop.